app/_landing_pages/konnect.yaml

```diff
@@ -273,110 +293,55 @@ rows:
+            config:
+              products:
+                  - konnect
+              works_on:
+                  - konnect
+              quantity: 10
   
+
   - header:
       type: h2
-      text: Support policies
+      text: How-to guides
     columns:
       - blocks:
-          - type: card
-            config:
-              title: "{{site.konnect_short_name}} compatibility"
-              description: |
-                See which browsers, software versions, tools, and applications {{site.konnect_short_name}} is compatible with.
-              cta:
-                text: View the compatibility policy 
-                url: /konnect-platform/compatibility/
-                align: end
-      - blocks:
-          - type: card
-            config:
-              title: "{{site.base_gateway}} third-party dependencies"
-              description: |
-                This reference lists all of the tested and supported versions of {{site.base_gateway}}'s third-party dependencies.
-              cta:
-                text: View the supported third-party dependencies 
-                url: /gateway/third-party-support/
-                align: end
-      - blocks:
-          - type: card
-            config:
-              title: "{{site.konnect_short_name}} release notes"
-              description: |
-                Learn about new {{site.konnect_short_name}} features.
-              cta:
-                text: View the changelog 
-                url: https://app.getbeamer.com/kongreleases/en
-                align: end
-  - columns:
-      - blocks:
-          - type: card
-            config:
-              title: "Kong vulnerability patching process"
-              description: |
-                Learn how Kong handles vulnerabilities or potential vulnerabilities in {{site.base_gateway}} or 
-                third-party code, and how to report any security issues.
-              cta:
-                text: View the vulnerability policy 
-                url: /gateway/vulnerabilities/
-                align: end
-      - blocks:
-          - type: card
-            config:
-              title: "{{site.konnect_short_name}} account"
-              description: |
-                Learn about the different {{site.konnect_short_name}} plans and account management.
-              cta:
-                text: View {{site.konnect_short_name}} account information 
-                url: /konnect-platform/account/
-                align: end
-      - blocks:
-          - type: card
+          - type: how_to_list
             config:
-              title: "{{site.konnect_short_name}} pricing"
-              description: |
-                Learn about {{site.konnect_short_name}} plans and pricing.
-              cta:
-                text: View {{site.konnect_short_name}} pricing 
-                url: https://konghq.com/pricing
-                align: end
-  
+              products:
+                  - gateway
+              works_on:
```

Comment:
Looks like this `works_on` isn't working as expected since it's directing me to /how-to/?products=gateway

Reply:
Hmm. It is working as configured (gateway how-tos + compatible with konnect) but now that I think about it, having only "gateway" as a product doesn't make sense anymore.